### ISSUE TITLE
Can't copy files form shared folder

### ISSUE THREAD

**Issue by ph55** [Parallels/vagrant-parallels#361]
When trying to copy any file from shared folder (`/vagrant`) on host machine:

```
cp /vagrant/filename.txt /vagrant/filename_copy.txt 
```

It fails with message:

```
cp: skipping file 'filename.txt', as it was replaced while being copied
```

I guess the reason is that shared folders constantly syncing files to host machine.

**Comment by ikari7789** [comment]
I can't copy anything from any shared folder onto anywhere on the guest machine.

```
$ cp /vagrant/testfile /home/vagrant/testfile
cp: skipping file ‘/vagrant/testfile’, as it was replaced while being copied
```

**Comment by mpdude** [comment]
Same here 👋🏻. Seems to be a (breaking) change from Parallels 15 -> Parallels 16; #362 might be a fix.

**Comment by stephenreay** [comment]
Ugh I've just hit this same issue. @romankulikov are you able to look at the referenced PR for this?

**Comment by legal90** [comment]
Hi all,

It's confirmed that this issue exists on all Vagrant VMs with Linux which have Parallels Tools v16 or higher installed. You can find more details in this comment: https://github.com/Parallels/vagrant-parallels/pull/362#issuecomment-695032658

As a workaround, please, ensure that you have Parallels Desktop v16 installed on your and Mac host (`prlctl --version`) and Parallels Tools v16 in your VM (`prltoolsd -V` on VM) and then add this line to your Vagrant file to fix it for the default shard folder:
```ruby
# Vagrant.configure("2") do |config|
<...>
config.vm.synced_folder "./", "/vagrant", mount_options: ["share", "noatime", "host_inodes"]
<...>
#
```
For custom `synced_folder` statements - just add `host_inodes` mount option, making it similar to the example above.

**Comment by legal90** [comment]
Hi all,
It seems that the issue was fixed in Parallels _[Guest]_ Tools v16.1.49141  (available in Parallels Desktop v16.1.1)
```
vagrant@ubuntu-18:~$ prltoolsd -V
Parallels Tools: 16.1.49141
```

Starting since this version, inodes are persistent as it was before v16 release, so there is no need to use `host_inodes` mount option anymore. 
But since Parallels Tools are bundled with the base box, it will take some time for box maintainers (like [bento](https://app.vagrantup.com/bento/), [generic](https://app.vagrantup.com/generic/) or [laravel](https://app.vagrantup.com/laravel/)) to rebuild their parallels boxes with latest version of guest tools.

To get it fixed for you right now, please do the following:
1. Upgrade your Parallels Desktop for Mac to the latest version (16.1.1 or higher): `Parallels Desktop` top menu -> `Check for Updates...`
2. Enable the auto-upgrade of Parallels Tools by adding this to your Vagrantfile (unless you already have it):
```ruby
config.vm.provider "parallels" do |prl|
  prl.update_guest_tools = true
end
```
3. Restart your VM, `vagrant reload`
